src/components/SelectedFilterOptionList/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./index.css";
+
+// It renders the selected filter options of a given state which tracks the selected filter options.
+// It is called in AdvancedFilters component
+const FilterOptionsList = ({ selectedOptions }) => {
+  return (
```

Comment:
Maybe a blank line before return?

Reply:
I tried: if I add extra space there, tests don't pass here. When I run yarn fix it deletes extra space.